features/environment.py

```diff
@@ -128,13 +129,18 @@ def _make_patroni_test_config(self, name, dcs, tags):
             config['tags'] = tags
 
         if dcs != 'etcd':
-            etcd = config.pop('etcd')
-            config['zookeeper'] = {'scope': etcd['scope'], 'session_timeout': etcd['ttl'],
-                                   'reconnect_timeout': config['loop_wait']}
-            if dcs == 'exhibitor':
-                config['zookeeper']['exhibitor'] = {'hosts': ['127.0.0.1'], 'port': 8181}
+            dcs_config = config.pop('etcd')
```

Comment:
I don't think it would confuse anyone if we add a comment stating the order the DC sections are picked and instruct the user to comment out those sections they don't want. It won't change the default behavior of picking etcd as well. And it will make the code in behave tests much less confusing, since there will be no need to pick up etcd section and transform it somehow to consul or zookeeper. I won't object those changes to be outside of this pull-request though.